framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -2204,12 +2212,64 @@ public ParameterizedExecutableType methodFromUse(
 
     if (tree.getKind() == Tree.Kind.METHOD_INVOCATION
         && TreeUtils.isMethodInvocation(tree, objectGetClass, processingEnv)) {
-      adaptGetClassReturnTypeToReceiver(methodType, receiverType);
+      adaptGetClassReturnTypeToReceiver(methodType, receiverType, tree);
     }
 
     return new ParameterizedExecutableType(methodType, typeargs);
   }
 
+  /**
+   * Apply capture conversion to the type arguments of a method invocation.
+   *
+   * @param typeVarMapping mapping from type variable to its argument
+   * @param declTypeVar list of type variable declarations
+   * @return a mapping form type variable to its captured type argument
+   */
+  private Map<TypeVariable, AnnotatedTypeMirror> captureMethodTypeArgs(
+      Map<TypeVariable, AnnotatedTypeMirror> typeVarMapping,
+      List<AnnotatedTypeVariable> declTypeVar) {
+    // TODO: This should happen as part of Java 8 inference.
```

Comment:
What change to the code will be made?  Will this method be eliminated, or is this comment about some part of the method body?